src/import/ImportWavPack.cpp

```diff
@@ -104,11 +105,12 @@ TranslatableString WavPackImportPlugin::GetPluginFormatDescription()
 std::unique_ptr<ImportFileHandle> WavPackImportPlugin::Open(const FilePath &filename, AudacityProject*)
 {
    char errMessage[100]; // To hold possible error message
-   int flags = OPEN_WVC | OPEN_FILE_UTF8 | OPEN_TAGS;
+   int flags = OPEN_WVC | OPEN_FILE_UTF8 | OPEN_TAGS | OPEN_DSD_AS_PCM | OPEN_NORMALIZE;
    WavpackContext *wavpackContext = WavpackOpenFileInput(filename, errMessage, flags, 0);
    
    if (!wavpackContext) {
       // Some error occured(e.g. File not found or is invalid)
+      wxLogDebug("WavpackOpenFileInput() failed on file %s, error = %s", filename, errMessage);
       return nullptr;
```

Comment:
Probably `wxLogError` is a better idea. `wxLogDebug` does not generate any user visible output really.

Reply:
I didn't really intend this for the user, especially because this gets triggered any time the WavPack importer is invoked to check whether some random file is importable. However, it might be useful for debugging if `libwavpack` refuses to open a file that it _should_ be able to.